src/yadmark/benchmark/external/apfelpy_utils.py

```diff
@@ -0,0 +1,312 @@
+"""
+    Utilities to help run APFEL++ benchmarks.
+"""
+import numpy as np
+
+# Q2 knots specs
+NQ = 50
+QMin = 1
+QMax = 200
+
+# Map observables names to APFEL++ methods
+MAP_HEAVINESS = {
+    "charm": 4,
+    "bottom": 5,
+    "top": 6,
+    "light": 0,  # TODO: Check combination
+    "total": 0,
+}
+
+
+def couplings(ap, pids, proc_type, obs_name):
+    """Return the corresponding coupling given a process type
+    and an observable.
+
+    Parameters
+    ----------
+    pids: int
+        PDG ID of the projectile
+    proc_type:
+        type of the DIS process
+    obs_name: str
+        name of the Yadism observable
+
+    Returns
+    -------
+    callable
+        a callable function that computes the coupling as a
+        function of the scale Q
+    """
+
+    # Effective charges
+    def _fBq(Q):
+        if proc_type == "EM":
+            # For Q=0 we only have electric charges
+            return ap.utilities.ElectroWeakCharges(0, False, pids)
+        return ap.utilities.ElectroWeakCharges(Q, False, pids)
+
+    def _fDq(Q):
+        if proc_type == "EM":
+            return [0.0]
+        return ap.utilities.ParityViolatingElectroWeakCharges(Q, False, pids)
+
+    # CKM matrix elements
+    def _fCKM(Q):
+        return ap.constants.CKM2
+
+    if proc_type == "CC":
+        coupling = _fCKM
+        if obs_name.startswith("g"):
+            raise ValueError("Yadism cannot compute polarised CC yet.")
```

Comment:
```suggestion
            raise ValueError("Apfel++ cannot compute polarised CC yet.")
```
Let's check here only things which are not implemented in Apfel++. 
We are planning to implement them soon, but not sure if Valerio is planning to add them as well. 
Yadism input checks should go here https://github.com/NNPDF/yadism/blob/master/src/yadism/input/